docs/landing.md

```diff
@@ -0,0 +1,46 @@
+# Spark Integration Hub K8s documentation
+
+Spark Integration Hub K8s is an open-source [Kubernetes charm](https://juju.is/docs/olm/charmed-operator) that enables the integration of the Charmed Apache Spark solution with existing charms (PostgreSQL, S3-integrator, etc.) or bundles (COS, Identity, etc.) and properly configure the corresponding option in the Spark ecosystem.
+
+The Spark Integration Hub K8s charm provides centralization for other charmed components and generates the configuration options for the Spark jobs that will be run using Charmed Spark tooling (e.g. Spark-client snap, spark8t library, charmed applications).
```

Comment:
```suggestion
The Integration Hub charm handles relations (integrations) with other charmed operators (PostgreSQL, S3-integrator, etc.) and bundles (COS, Identity, etc.), manages service accounts, updates from integrated charms, and prepares configuration options for Spark jobs.
```
That handles the second question: what the product **does**?

Reply:
Same, used your (un-credited) suggestion